cypress/e2e/po/components/generic-table.po.ts

```diff
@@ -0,0 +1,24 @@
+import ComponentPo from '@/cypress/e2e/po/components/component.po';
+import ListRowPo from '@/cypress/e2e/po/components/list-row.po';
+
+export default class GenericTablePo extends ComponentPo {
```

Comment:
I don't think this file is necessary as it's the same as using `SortableTablePo` or am I missing something?

Reply:
The search of strings in the cluster detail tables fail with the `rowWithName` regex. I was testing this, I reverted it back to `rowWithPartialName`